### ISSUE TITLE
Problem in CapG_Model.py

### ISSUE THREAD

**Issue by czhxiaohuihui** [metro-smiles/CapG_RevG_Code#1]
def forward(self, input, hidden, flag):
		if flag == 'level_1': # Passing image features and stars for the first GRU of every sentence - Sentence RNN
			ip = self.img_encoding(input) # .view(1, 1, -1)
			ip = ip.view(1, 1, -1)
			#hidden = torch.cat([hidden, str_ip], 2)
			output, hidden = self.gru_cont(ip, hidden)
			output = self.softmax(self.out_cont(output[0])) # Obtain the labels of whether to continue or stop
		elif flag == 'level_1': # Passing word embeddings - Word RNN
			output = self.embedding(input).view(1, 1, -1)
			#print('Processed Input Embedding')
			output = F.relu(output)
			output, hidden = self.gru_text(output, hidden)

output = self.softmax(self.out_text(output[0]))

Are there  two 'level_1'  in the 'if--elif---' ????

**Comment by metro-smiles** [comment]
Good Point! Updated the code
